scss/upload/_layout.scss

```diff
@@ -210,8 +216,9 @@
     }
 
     .k-file-name-size-wrapper {
-        vertical-align: middle;
-        margin-left: 1em;
+        display: block;
+        margin-left: calc(#{$padding-x} + #{40px});
+        margin-right: calc(#{$padding-x} + #{16px + #{$padding-x-sm}});
```

Comment:
The icon size is available through the `$icon-size` variable, so using that will explain the intent.
The uploaded file image size seems computed from the `$padding-y-lg` and the width of `.k-file-extension-wrapper`. Using variables will continue to work after their values are changed, for example in a custom theme.